dom-creation/dom-create-quiz-notes.md

```diff
@@ -0,0 +1,48 @@
+# dom-creation-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- Does the `document.createElement()` method insert a new element into the page?
+  yes, it creates a new element with the specified tag name
```

Comment:
It does create the new element, but note that it does not add it the the DOM tree (page) until you use the document.appendChild() method to hook it up to an existing element in the DOM tree